src/types/api.js

```diff
@@ -142,6 +142,7 @@
  *
  * @typedef Group
  * @prop {string} id
+ * @prop {string} groupid
```

Comment:
I verified that `groupid` is a field that exists in the backend response. The difference between `id` and `groupid` is that `groupid` is controlled by the API consumer that creates a group, whereas `id` is a purely server-determined ID. IIRC `groupid` is only used in the LMS context.

Reply:
I presume this is `groupid`, not `groupID` because it's named after the API response property.

Reply:
You're correct.